src/style.css

```diff
@@ -1,3 +1,89 @@
+* {
+  --text-height: 15px;
+  --text-gap: 20px;
+  --img-width: 40px;
+  --img-height: 40px;
+  --header-height: 60px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+a,
+li {
```

Comment:
<img width="1088" alt="image" src="https://github.com/user-attachments/assets/785ffb7d-5987-4446-b82c-8aff7f830b04">

Reply:
> <img alt="image" width="1088" src="https://private-user-images.githubusercontent.com/86789161/357022573-785ffb7d-5987-4446-b82c-8aff7f830b04.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0ODA4NTYsIm5iZiI6MTcyMzQ4MDU1NiwicGF0aCI6Ii84Njc4OTE2MS8zNTcwMjI1NzMtNzg1ZmZiN2QtNTk4Ny00NDQ2LWI4MmMtOGFmZjdmODMwYjA0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEyVDE2MzU1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlOWU3M2VkZGY2ZjI2NTMwNTkzODI0ZGQ3OWNjYWNjYTgzNTA2YzBjYmMwMDc4MWMyNWE4ZDllM2IwYjIxMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DUBJefUB_wzxBVylxJVg3x6RZzLF--93hCCxlo66hz8">


Not fixed